isis/src/clementine/apps/clemhirescal/clemhirescal.xml

```diff
@@ -9,14 +9,17 @@
       camera. It does so through the use of some constants based on offset and
       flat images based on filter type. There are six possible offsets, with the
       resultant change being anywhere between eight and fifty to the DN. There
-      are four flat images, one for each of the four possible filters. 
-      Information for this program came from JGR Vol 108, A radiometric 
+      are four flat images, one for each of the four possible filters.
+      Information for this program came from JGR Vol 108, A radiometric
       calibration for the Clementine Hires camera: Robinson, Malaret, White.
```

Comment:
@AustinSanders Can you add the persistent link for the paper? https://doi.org/10.1029/2000JE001241